test/test_kmer_collection.cc

```diff
@@ -43,10 +43,60 @@ TEST_CASE("Basic functionalities of kmer_collection", "[kmer_collection]") {
     REQUIRE(itBegin != itEnd);
   }
 
-  SECTION("comparison for equality") {
+  SECTION("Comparison for equality 1") {
     REQUIRE(itBegin == itBegin);
     REQUIRE(itEnd == itEnd);
```

Comment:
So test cases and sections should always describe:

1. What is being tested
1. The argument(s)
1. The expected outcome of the test